rest-api/tools/auth_setup.sh

```diff
@@ -50,6 +50,7 @@ fi
 
 REPO_ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )"
 source ${REPO_ROOT_DIR}/tools/setup_vars.sh
+TMP_GEOCODE_INFO_FILE=${TMP_DIR}/geocode_key.json
```

Comment:
It might be redundant, but we may want to explicitly rm this in cleanup, like we do for TMP_CREDS_FILE, etc.